apps/emqx_conf/src/emqx_conf_schema_inject.erl

```diff
@@ -29,6 +29,7 @@ schemas(Edition) ->
         authz() ++
         shared_subs(Edition) ++
         bridges(Edition) ++
+        channel_customizations() ++
```

Comment:
this injection has little to do with schema, maybe use a new injection mechanism.
e.g. `limiter_override_cb` as persistent term key and call it directly from emqx_mt app